src/utilities/configuration-utilities.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * Configuration Utilities
+ *
+ * Contains alias functions
+ */
+
+use Nectary\Configuration;
+
+/**
+ * Handle configuration call without knowing about how
+ * the Configuration object works
+ */
+function config( $key, $default = null ) {
```

Comment:
Added this as a nice shorthand that will get the singleton instance for you since writing `Configuration::get_instance()->get()` would be really annoying.